index.php

```diff
@@ -5,21 +5,38 @@
     <title>Netflix Series Collection</title>
 </head>
 <body>
+<h2>Complete all fields to add a new series to your collection</h2>
+<!-- -->
+<form method="post" action="adddata.php">
+    Series Title:
+    <input type="text" name="seriestitle">
+    Date Started:
+    <input type="date" name="datestarted">
+    Date Ended:
+    <input type="date" name="dateended">
+    Number of seasons:
+    <input type="number" name="noofseasons" min="1" max="50">
+    Number of episodes:
+    <input type="number" name="noofepisodes" min="1" max="300">
+    Director:
+    <input type="text" name="director">
+    <input type="submit" value="submit">
+</form>
+
 <?php
-$db = new PDO('mysql:host=db;dbname=netflix_series','root','password');
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,PDO::FETCH_ASSOC);
-$query = $db-> query('SELECT `id`,`title`,`start_date`,`end_date`,`number_of_seasons`,`number_of_episodes`,`director` FROM `favourites`');
+//As function connect_to_netflix_series_db has no parameters it will run here
```

Comment:
Fine for your notes but something to get in the habit of removing from production code

Reply:
All comments removed